app/views/webhooks/index.html.erb

```diff
@@ -12,14 +12,24 @@
 
   <dl>
     <% Samson::Integration::SOURCES.each do |source| %>
-      <dt><%= source.titleize %>:</dt><dd><%= link_to_url(send("integrations_#{source}_deploy_url", @project.token)) %></dd>
+      <dt>
+        <%= source.titleize %>:</dt><dd><%= link_to_url(send("integrations_#{source}_deploy_url", @project.token)) %>
+        <% if source == 'github' && token = Integrations::GithubController.secret_token %>
+          <br/>
+          <% if current_user.super_admin? %>
```

Comment:
Since we have the name of the GitHub repo, maybe we could also have a link to the "settings" page of that repo, telling a user how to set up the webhook?